src/theme/theme.tsx

```diff
@@ -19,7 +18,7 @@ const ThemeProvider = ({
   theme: ThemeName;
   children: React.ReactNode;
 }) => (
-  <StyledThemeProvider theme={themes[name]}>
+  <StyledThemeProvider theme={{ name, ...themes[name] }}>
```

Comment:
@serdec @gjones @fneves I had a doubt
Since we are allowing the theme option to be passed as a prop optionally and getting the default name from current system I implemented it like this
Wanted to know if this would be a good option or add it in the official themes token?
Would like to know your inputs before proceeding and writing tests
Currently it is working as desired . If all is good I can add a video showing the functionality

Reply:
let's try adding the name of the theme in the theme @gjones 

another alternative could be to save it in the local storage and retrieve it from there, but i think the former option is cleaner

Reply:
Got it, I've pushed the tokens, here's the [commit](https://github.com/ClickHouse/click-ui/commit/2ff3c820ba1257990921d7491a64c00936eda811)